packages/components/pie-webc-core/package.json

```diff
@@ -10,7 +10,7 @@
     "build": "run -T vite build",
     "lint:scripts": "run -T eslint .",
     "lint:scripts:fix": "run -T eslint . --fix",
-    "test": "run -T vitest run --config ../../../vite.config.js",
+    "test": "run -T vitest run --config ./vite.config.js",
```

Comment:
@siggerzz I also pointed to the pie-webc-core vite config here. I don't know why it was pointing to the core one. Was there a specific reason?